sample-components/rollup.config.js

```diff
@@ -0,0 +1,19 @@
+// Rollup config for consuming some npm modules in MagicScript
+import babel from 'rollup-plugin-babel';
+import resolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+
+export default {
+  external: ['uv', 'lumin'],
```

Comment:
We should at least add `ssl` here so rollup doesn't complain.  It would be visible if someone ran the rollup step manually.

Reply:
done